src/print-pretty-error.ts

```diff
@@ -1,8 +1,10 @@
 // Dependencies
 import chalk from './chalk';
 
-// Print a pretty error message
+// Print a pretty error message and return just the message
 export default function printPrettyError(message: string) {
 	console.error(`${chalk.inverse.red.bold(' ERROR ')}\n`);
 	console.error(chalk.red(`> ${message.split('\n').join('\n> ')}\n`));
+
+	return message;
```

Comment:
See warning from Codecov below. This line needs testing.

Reply:
I added a test that checks the response value.